### PR TITLE
now displaying gift cards in order delivery detail

### PR DESCRIPTION
**Pull request by jasperboyd**

### CODE REVIEW

**Comment by rmarchandTen24** [comment]
can we go over this smartlist? We may be able to make this simpler

File: model/entity/OrderDelivery.cfc [hunk 1]
```diff
@@ -84,6 +84,18 @@ component displayname="Order Delivery" entityname="SlatwallOrderDelivery" table=
 		return totalDelivered;
 	}
 
+	public any function getOrderDeliveryGiftCardSmartList() {
+		var giftCardSmartList = getService("HibachiService").getGiftCardSmartList();
```
Comment (gregmoser) [inline model/entity/OrderDelivery.cfc]: You should be getting this NOT for getService("hibachiService") but it should be getService("GiftCardService")

Comment (jasperboyd) [inline model/entity/OrderDelivery.cfc]: Doesn't the get#entity#smartlist() come from hibachi service? or is this more of a convention?

Comment (rmarchandTen24) [inline model/entity/OrderDelivery.cfc]: yes true but going through the giftcardservice is in preparation for an override

**Pull request merged**